setup.py

```diff
@@ -18,6 +18,9 @@ def package_files(directory):
                 'modules/mavproxy_magical/data/*.obj',
                 'modules/mavproxy_fieldcheck/*.txt',
                 'tools/graphs/*.xml',
+                'modules/mavproxy_nokov/nokov/*',
+                'modules/mavproxy_nokov/nokov/lib/*',
```

Comment:
why add in these subdirectories? they don't seem to exist

Reply:
Yes, we did not submit the nokovsdk.

Reply:
The SDK has been committed.